power/SaifReader.cc

```diff
@@ -191,6 +192,7 @@ SaifReader::setNetDurations(const char *net_name,
                    tc,
                    activity);
         power_->setUserActivity(pin, activity, duty, PwrActivityOrigin::saif);
+        std::cout << "SAIF Annotated pin: " << net_name << " tr. count: " << tc <<  " activity: " << activity << " duty: " <<  duty << std::endl;
```

Comment:
This should use a debug logger instead. Are we even supporting SAIF at this point anyways?